renderMath.js

```diff
@@ -223,7 +231,7 @@ function renderMath(parentElement = "") {
                     var newBR = createHTMLNodesFromString("\n\\\\\n");
                     oldBR.replaceWith(...newBR);
                 }
-                oldSpanAlignedEquations.innerHTML = oldSpanAlignedEquations.innerHTML.replace(/=/g, "&=").replace(/\-\>/gm, "&->").replace(/\-&gt;/gm, "&->");
+                oldSpanAlignedEquations.innerHTML = oldSpanAlignedEquations.innerHTML.replace(/=/g, "&=");
                 var newSpanAlignedEquations = createHTMLNodesFromString("\\begin{align}" + oldSpanAlignedEquations.innerHTML + "\\end{align}");
```

Comment:
On your end, change the `align` to `aligned` inside the MathJax strings, let me know if it renders correctly now
```suggestion
                var newSpanAlignedEquations = createHTMLNodesFromString("\\begin{aligned}" + oldSpanAlignedEquations.innerHTML + "\\end{aligned}");
```

However if that does not work, then try:

```suggestion
                var newSpanAlignedEquations = createHTMLNodesFromString("\\eqalign{" + oldSpanAlignedEquations.innerHTML + "}");
```

And if that doesn't work then give up with 2.7 and we're sticking with 3.x because who really needs to copy formulas from our study guide.